app/population/middleware/set-breadcrumb.js

```diff
@@ -0,0 +1,26 @@
+const { format, parseISO } = require('date-fns')
+
+const { DATE_FORMATS } = require('../../../config/index')
+
+function setBreadcrumb(req, res, next) {
+  const { date, locationName } = req
+
+  res.breadcrumb({
```

Comment:
Because `breadcrumb` returns the `res` object these can be chained to:

```javascript
res
  .breadcrumb({
    text: 'Population Overview',
    href: `/population/week/${date}`,
  })
  .breadcrumb({
    text: locationName,
    href: '',
  })
  .breadcrumb({
    text: format(parseISO(req.date), DATE_FORMATS.WITH_DAY),
    href: req.population ? req.baseUrl : '',
  })
```

Reply:
Done!